tesseract-server/src/handlers/logic_layer/detection.rs

```diff
@@ -0,0 +1,187 @@
+use actix_web::{
+    AsyncResponder,
+    FutureResponse,
+    HttpRequest,
+    HttpResponse,
+    Path,
+};
+use failure::{Error, format_err};
+use futures::future::{self, Future};
+use lazy_static::lazy_static;
+use log::*;
+use serde_derive::{Serialize, Deserialize};
+use serde_qs as qs;
+use std::convert::{TryFrom, TryInto};
+use tesseract_core::{Schema, Cube};
+use tesseract_core::format::{format_records, FormatType};
+use tesseract_core::Query as TsQuery;
+
+use crate::app::AppState;
+use crate::handlers::aggregate::AggregateQueryOpt;
```

Comment:
This route will need to use a different api than the other aggregate handler. Once again, sorry for not specifying better before.

This will be good to document as part of the pr also.

If you check canon's logic layer readme, you'll see that some of the keys in the query string are the same, but that they are all comma separated lists in a string instead of the cube api's list notation `[]`.

Also cuts are processed as key-value pairs, instead of as `cuts[]={member}`. This is where you'll have to use:
```
#[derive(Deserialize)]
struct LogicLayerQueryOpt {
  measure: String,
  year: Year,
  #[serde(flatten)]
  cuts: std::collections::HashMap,
}
```

And for now, it's fine to resolve to just one cube, but in the future we'll need to map multiple measures to their cube.